tests/test_other.py

```diff
@@ -5980,3 +5980,118 @@ def test_override_environment(self):
               assert ('environment is %s? false' % other) in seen, seen
           print '-- verified proper env is shown'
 
+  ############################################################
+  # Function eliminator tests
+  ############################################################
+  def normalize_line_endings(self, input):
+    return input.replace('\r\n', '\n').replace('\n\n', '\n').replace('\n\n', '\n')
+
+  def get_file_contents(self, file):
+    file_contents = ""
+    with open(file) as fout:
+      file_contents = "".join(fout.readlines())
+
+    file_contents = self.normalize_line_endings(file_contents)
+
+    return file_contents
+
+  def function_eliminator_test_helper(self, input_file, expected_output_file, use_hash_info=False):
```

Comment:
The test_function_eliminator_*_with_hash_info tests set this to true. But it looks like we're not passing that to the eliminate-duplicate-functions.js file - the effect is benign since then we just perform DFE rather than just the elimination using the hash. But for correctness, we should just perform the elimination using the hash info. Updated in the next iteration.